cinder/Changelist.md

```diff
@@ -1,4 +1,3 @@
 | Tags | Image version | Changes | Build date | Build type |
 | ---- | ------------- | ------- | ---------- | ---------- |
-| 16.2.0-6 | 4.4.0 | Initial version of RHOSP16.2.0, base image 16.2.0-61, storops 1.2.7 | 2021-09-17 | Manual |
```

Comment:
Why delete the old history here?

Reply:
This is the first version for RHOSP17, so remove entries for 16. I'm following the practice in previous versions. You can refer to branches for other version.